lib/common/output.c

```diff
@@ -24,6 +24,8 @@ pcmk__output_formatters(void) {
 
 void
 pcmk__output_free(pcmk__output_t *out) {
+    CRM_ASSERT(out != NULL);
+
```

Comment:
For free functions, it's better to just no-op like free(NULL)

This is particularly helpful when freeing higher-level types that have a member of this type, which may or may not be set when the higher-level free function is called

Reply:
Yeah, good idea.  I think I got them all between this patch and the individual free_priv functions elsewhere.